testing/tests/DevExpress.ui/collectionWidget.tests.js

```diff
@@ -1624,10 +1624,13 @@ module('keyboard navigation', {
 
         $element.focusin();
         $item.trigger('dxpointerdown');
+
         this.clock.tick();
+
         keyboard.keyDown('right');
-        assert.equal(instance.option('selectedIndex'), 3, 'selectedIndex is correctly');
-        $item = $($items.get(3));
+        assert.equal(instance.option('selectedIndex'), 0, 'selectedIndex is correctly');
```

Comment:
```suggestion
        assert.strictEqual(instance.option('selectedIndex'), 0, 'selectedIndex is correct');
```

Reply:
Test was updated in this commit [25bf9f775185867becc632f8384161cc5d472868]. Thanks!